app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -139,6 +140,9 @@ class IssueUpdateBuilder {
   }
 
   String get issueUpdateComment {
+    if (statistic.flakyBuilds!.length + statistic.succeededBuilds!.length < kFlayRatioBuildNumberList) {
```

Comment:
Maybe we just skip if this is below threshold? on one hand i know printing this comment let us know script is still working, but this comment is not really useful for triaging. I am neutral on this.

Reply:
It makes sense to me to skip the comment. Updated.